src/map/script-fun.cpp

```diff
@@ -2175,6 +2175,42 @@ void builtin_pvpoff(ScriptState *st)
     }
 }
 
+static
+void builtin_pvp(ScriptState *st)
+{
+    dumb_ptr<map_session_data> sd = script_rid2sd(st);
+    sd->state.pvpon = conv_num(st, &AARG(0));
+
+    if (sd->state.pvpon)
+        clif_displaymessage(sd->sess, "Pvp: On"_s);
```

Comment:
why display messages on the pvp/killer/killable builtins? if the dev want to display a message then they'll manually add one